Dockerfile

```diff
@@ -19,4 +19,4 @@ COPY application/ /data/
 RUN chown -R www-data:www-data \
     console/runtime/
 
-CMD ["/data/yii", "batch/full"]
+CMD ["/data/run.sh"]
```

Comment:
After closer inspection, I'd prefer to put this back the way it was. The run.sh referenced in docker-compose.yml and codeship-services.yml is for id-broker not id-sync. If it becomes necessary to make run.sh more complex, then it might seem worth referencing a script. There are two minor disadvantages to using a script, (1) the confusion of broker's `run.sh` and id-sync's `run.sh` persists, and (2) running a script in CMD potentially leads to issues that arise from the main process not being PID 1.

Reply:
Thanks